File: android/src/main/kotlin/com/benjamin/horner/flutter_acs_card_reader/enums/APDUReadResponseEnum.kt
package com.benjamin.horner.flutter_acs_card_reader

enum class APDUReadResponseEnum {
    SUCCESS,
    NO_EF_SELECTED,
    OFFSET_GREATER_THAN_EF,
    OFFSET_PLUS_LENGTH_GREATER_THAN_EF,
    FILE_ATTRIBUT_INTEGRITY_ERROR,
    STORED_DATE_INTEGRITY_ERROR
}

File: ios/Classes/Enums/APDUReadResponseEnum.swift
//
//  APDUReadResponseEnum.swift
//  flutter_acs_card_reader
//
//  Created by Benjamin Horner on 23/11/2023.
//

import Foundation

enum APDUReadResponseEnum {
    case success
    case offsetPlusLengthGreaterThanEF
    case offsetGreaterThanEF
    case fileAttributeIntegrityError
    case storedDataIntegrityError
}
